### DISCUSSION TITLE
Open youtube link error

### DISCUSSION THREAD

**Discussion by WhyredXCB** [termux/termux-app#3281] [Q&A] [answered]
Does anyone know how to solve termux-open-url cannot open youtube link?
Been trying to open other link and it's worled except for youtube link
I'll leave an screenshot about the error here https://ibb.co/7SkY19d

**Comment by RenanSD007** [comment]
Try enabling the "Draw over other apps" permission

**Reply by WhyredXCB** [reply, marked as answer]
Thanks, for replying, I already found the way to fix it. Need to set default apps to open the youtube itself to make it work cause I haven't set the default apps before
